src/test/java/org/cyclops/integrateddynamics/core/evaluate/variable/TestStringOperators.java

```diff
@@ -780,4 +780,18 @@ public void testInvalidInputTypeNamedName() throws EvaluationException {
         Operators.NAMED_NAME.evaluate(new IVariable[]{DUMMY_VARIABLE});
     }
 
+    /**
+     * ----------------------------------- STRING_ERROR -----------------------------------
+     */
+
+    @Test(expected = EvaluationException.class)
+    public void testSymbolsError() throws EvaluationException {
+        Operators.STRING_ERROR.evaluate(sregex);
+    }
+
+    @Test(expected = EvaluationException.class)
+    public void testSpacesError() throws EvaluationException {
+        Operators.STRING_ERROR.evaluate(shelloWorld);
+    }
+
```

Comment:
Could you also add tests for checking too many, too few, and incorrectly typed inputs? (just like [the other tests](https://github.com/CyclopsMC/IntegratedDynamics/blob/0d84dfeea8d48357d929f4fdf1e6fc5cfc75070c/src/test/java/org/cyclops/integrateddynamics/core/evaluate/variable/TestStringOperators.java#L158-L171))